boards/raytac/mdbt50q_cx_40_dongle/doc/index.rst

```diff
@@ -0,0 +1,294 @@
+                -B smp_svr.signed.bin -- --key mcuboot/root-rsa-2048.pem
+
+#. Flash the application to the MCUboot serial port using ``mcumgr``:
+
+   .. code-block:: console
+
+      mcumgr --conntype=serial --connstring='dev=/dev/ttyACM0,baud=115200' \
+             image upload -e smp_svr.signed.bin
+
+#. Reset the device:
+
+   .. code-block:: console
+
+      mcumgr --conntype=serial --connstring='dev=/dev/ttyACM0,baud=115200' reset
+
+You should now be able to scan for Bluetooth devices using a smartphone or
+computer. The device you just flashed will be listed with ``Zephyr`` in its
+name.
+
+.. note::
+
+   This board supports building other Zephyr applications for flashing with
+   MCUboot in this way also. Just make sure :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT`
+   is set when building your application. For example, to compile blinky for
+   loading by MCUboot, use this:
+
+   .. zephyr-app-commands::
+      :app: zephyr/samples/basic/blinky
+      :board: raytac_mdbt50q_cx_40_dongle/nrf52840
+      :build-dir: blinky
+      :goals: build
+      :gen-args: -DCONFIG_BOOTLOADER_MCUBOOT=y
+
+   You can then sign and flash it using the steps above.
+
+Option 3: Using an External Debug Probe
+---------------------------------------
+
+If you have one, you can also use an external :ref:`debug probe <debug-probes>`
+to flash and debug Zephyr applications, but you need to solder an SWD header
+onto the back side of the board.
+
+For Segger J-Link debug probes, follow the instructions in the
+:ref:`nordic_segger` page to install and configure all the necessary
+software. Further information can be found in :ref:`nordic_segger_flashing`.
+
+Locate the DTS file: :``boards/raytac/raytac_mdbt50q_cx_40_dongle_nrf52840.dts``.
+This file requires a small modification to use a different partition table.
+Edit the include directive to include "fstab-debugger" instead of "fstab-stock".
+
+In addition, the Kconfig file in the same directory must be modified by setting
+``BOARD_HAS_NRF5_BOOTLOADER`` to be default ``n``, otherwise the code will be
+flashed with an offset.
+
+Then build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :zephyr:code-sample:`blinky` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: raytac_mdbt50q_cx_40_dongle/nrf52840
+   :goals: build flash
+
+Observe the LED on the board blinking.
+
+Debugging
+=========
+
+The ``raytac_mdbt50q_cx_40_dongle/nrf52840`` board does not have an on-board J-Link debug IC
+as some nRF5x development boards, however, instructions from the
+:ref:`nordic_segger` page also apply to this board, with the additional step
+of connecting an external debugger.
+
+Testing the LEDs and buttons on the Raytac MDBT50Q-CX-40 Dongle
+***************************************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
```

Comment:
2 samples but you're only mentioning 1 :)

Reply:
I’ll update this in the next commit.